packages/nvidia_nat_strands/tests/test_strands_llm.py

```
@@ -0,0 +1,434 @@
+
+    def test_nim_compatible_openai_model_format_request_message_content_other_types(self):
+        """Test NIMCompatibleOpenAIModel.format_request_message_content handles other content types."""
+        # Test that non-reasoningContent types would fall back to parent
+        text_content = {"text": "Hello world"}
+
+        # The method should fall back to parent implementation for non-reasoning content
+        content = text_content
+
+        if "reasoningContent" in content:
+            # This should not be executed for text content
+            raise AssertionError("reasoningContent handling should not be triggered for text content")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)